h2o-core/src/main/java/water/rapids/ast/prims/reducers/AstCumMax.java

```diff
@@ -3,9 +3,7 @@
 
 public class AstCumMax extends AstCumu {
   @Override
-  public int nargs() {
-    return 1 + 1;
-  }
+  public int nargs() { return 1 + 2; }
```

Comment:
axis is now a new arg for AstCumu, which is why i changed it from 1+1 to 1+2
Are you saying its not needed? Im not sure myself...